build.gradle

```diff
@@ -386,9 +389,11 @@ applicationDefaultJvmArgs = [
 ]
 
 run {
-  args project.hasProperty("ethsigner.run.args") ? project.property("ethsigner.run.args").toString().split("\\s+") : []
+  args project.hasProperty(repositoryName.toLowerCase() + ".run.args") ? project.property(repositoryName.toLowerCase() + ".run.args").toString().split("\\s+") : []
   doFirst {
-    applicationDefaultJvmArgs = applicationDefaultJvmArgs.collect{it.replace('ETHSIGNER_HOME', "$buildDir/ethsigner")}
+    applicationDefaultJvmArgs = applicationDefaultJvmArgs.collect {
+      it.replace(repositoryName.toUpperCase() + "_HOME", "$buildDir/" + repositoryName.toUpperCase())
```

Comment:
the second argument should be lower case i.e. `"$buildDir/" + repositoryName.toLowerCase()))`

Reply:
done